avocado/core/loader.py

```diff
@@ -691,8 +690,7 @@ def ignore_broken(klass, uid, params=None):
                                                 subtests_filter)
             else:
                 if os.access(test_path, os.X_OK):
-                    return self._make_test(test.SimpleTest,
-                                           pipes.quote(test_path))
+                    return self._make_test(test.SimpleTest, test_path)
```

Comment:
Actually I could not determine the use case that led to the inclusion of the `pipes.quote()`. But it's clearly causing a regression. I hope @ldoktor can help me to clarify.